### DISCUSSION TITLE
Is this able to be used in an emulator?

### DISCUSSION THREAD

**Discussion by ExperiencersInternational** [RedShyGuy/Vapecord-ACNL-Plugin#106] [Q&A] [answered]
Hi there, I'd like to ask whether this is able to be used in an emulator, I watched a video on this recently and it looked cool.

If so, what would I need to do to get it working on Citra?

**Comment by RedShyGuy** [comment, marked as answer]
No you need Luma running for it to work, that only works on the 3DS itself

**Reply by ExperiencersInternational** [reply]
Oof that's a shame, thank you for letting me know

**Reply by Thevasy** [reply]
I was searching for this answer too. The Citra website say that for you to run Luma's mods you just need to estructure the folders and put on it the files necessary to run the mod. But I'm struggling with the files organization. I don't know where I must place each file because I don't know how it exactly works. I just readed  all pages of Lumas instructions, but I think I need help from someone that really understands how it works. Could you help me?

**Reply by ExperiencersInternational** [reply]
@Thevasy since I've posted this, I have downloaded this through Universal Updater.

The only reason why this doesn't work on emulator is the fact Citra has zero support for Luma plugins (that's why stuff like CTGP-7 have to have forks for Citra users to try it out, since Citra only has Luma mods support).

Would be nice if @citra-emu would allow people to use Luma plugins
